core/pom.xml

```diff
@@ -39,7 +39,7 @@ THE SOFTWARE.
 
   <properties>
     <staplerFork>true</staplerFork>
-    <stapler.version>1.254.2</stapler.version>
+    <stapler.version>1.255-null-1-SNAPSHOT</stapler.version>
```

Comment:
Pending JEP-305 integration, please use timestamped snapshots exclusively.

Reply:
I'm fairly sure I've observed problems in the past with Stapler, as we only explicitly specify two out of six components, and get the others transitively. Since the dependencies are only defined as `-SNAPSHOT`, we might end up with mismatched components.

So I'm doing one better: I have a unique version suffix. Probably should have called it `stapler-PR149` but whatever.

Reply:
While I'm strictly speaking losing some of the reproducibility, I gain massive laziness points (no need to update downstream PR on upstream update, just rebuild).

Reply:
Pending JEP-305, the correct way to handle Stapler is to declare the version of each component separately.

Reply:
This was just FYI while you had snapshot dependencies, no longer relevant until next time.